pkg/api/dirs_test.go

```diff
@@ -361,28 +303,28 @@ Disallow: /`),
 			validateIsPermanentRedirect := func(t *testing.T, fromPath, toPath string) {
 				t.Helper()
 
-				expectedResponse := fmt.Sprintf("<a href=\"%s\">Permanent Redirect</a>.\n\n", bzzDownloadResource(resp.Reference.String(), toPath))
+				expectedResponse := fmt.Sprintf("<a href=\"%s\">Permanent Redirect</a>.\n\n",
+					bzzDownloadResource(resp.Reference.String(), toPath))
 
-				jsonhttptest.Request(t, client, http.MethodGet, bzzDownloadResource(resp.Reference.String(), fromPath), http.StatusPermanentRedirect,
+				jsonhttptest.Request(t, client, http.MethodGet,
+					bzzDownloadResource(resp.Reference.String(), fromPath),
+					http.StatusPermanentRedirect,
 					jsonhttptest.WithExpectedResponse([]byte(expectedResponse)),
 				)
 			}
 
-			validateBzzPath := func(t *testing.T, fromPath, toPath string) {
+			validateAltPath := func(t *testing.T, fromPath, toPath string) {
```

Comment:
do we need the rename? I think `valdiateBzzPath` captures it no?

Reply:
This function validates that the file served on both the 'fromPath' and 'toPath' is the same (for redirects).

I named it 'AltPath' for 'alternate path'. I think this is better no?